set-mongo/src/main/java/eu/europeana/set/mongo/service/PersistentUserSetService.java

```diff
@@ -22,6 +23,15 @@
 	 */
 	public abstract PersistentUserSet getByIdentifier(String identifier);
 	
+	/**
+	 * This method checks if a user set with provided type and creator already exists in
+	 * database
+	 * @param type
+	 * @param creator
+	 * @return true if exists
+	 */
+	public abstract boolean isTypeAndCreatorExisting(String type, Agent creator);
```

Comment:
please refactor to getBookmarksFolder(String creatorId)

Reply:
refactored to public boolean getBookmarksFolder(String type, String creatorId)

Reply:
type is also required as parameter, because we also check it for update method

Reply:
the type is not needed, because only the BookmarksFolers are unique